presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -346,7 +346,7 @@ public synchronized void createTable(
             ConnectorSession session,
             Table table,
             PrincipalPrivileges principalPrivileges,
-            Optional<Path> currentPath,
+            Optional<Path> currentPath, // unpartitioned table only
```

Comment:
Should we consider renaming the variable to show it is for unpartitioned table only? Maybe `currentPathForUnpartitionedTable` so that we don't need a comment?

Reply:
It's a difficult call since this variable name also looks too long.

Reply:
What about `currentPathIfUnpartitioned`?

Reply:
@rschlussel : Thanks for the suggestion! This sounds reasonable. Do you mind apply it in an separate patch? (we should also rename other `Optional<Path> currentPath`, including class fields )